File: C++/W3Schools/14_Switch/14_Switch.cpp

// C++: W3Schools - Main Tutorial
// Section 14: Switch

#include <iostream>
using namespace std;

int main() {

  // BASICS
  int day = 4;
  switch (day) {                                                    // Switch statements compare a chosen value to a set of cases and execute all those that return true.
    case 6:
      cout << "1. " << "Today is Saturday" << endl;
      break;                                                        // If we only want to execute the first true instance, we can use the break keyword to stop the execution.
    case 7:
      cout << "2. " << "Today is Sunday" << endl;
      break;
    default:                                                        // If none of the cases are true, it will execute the default case.
      cout << "3. " << "Looking forward to the weekend!" << endl;
  }

  return 0;
}

File: W3Schools/Backend/Java/Main_Tutorial/14_Switch.java

// Java: W3Schools - Main Tutorial
// Section 14: Switch


public class Main {
  public static void main(String[] args) {

    int day = 4;

    // We can use switch statements to select one of many code blocks to be executed:
    switch (day) {
      case 6:
        System.out.println("Today is Saturday");
        break;  // The break keyword is used to halt execution of the switch statement.
      case 7:
        System.out.println("Today is Sunday");
        break;
      default:  // The default keyword specifies some code to run if there is no case match.
        System.out.println("Looking forward to the Weekend");
    }

  }
}
